runtime/vm/ModularityHashTables.c

```diff
@@ -189,3 +211,54 @@ findModuleInfoForModule(J9VMThread *currentThread, J9ClassLoader *classLoader, J
 	targetPtr = (J9ModuleExtraInfo *)hashTableFind(classLoader->moduleExtraInfoHashTable, (void *)&moduleInfo);
 	return targetPtr;
 }
+
+BOOLEAN
+didPackageExistInJAVA8(J9VMThread *currentThread, J9UTF8 *pkgName)
+{
+	J9JavaVM *vm = currentThread->javaVM;
+	BOOLEAN noFail = TRUE;
```

Comment:
Variable name like `succeed` in positive might be clearer.
In addition, can we add tests to validate that Java 8 packages are allowed but Java 9 newly added packages are not?